services/core-web/src/customAxios.js

```diff
@@ -3,30 +3,20 @@ import { notification } from "antd";
 import { isEmpty } from "lodash";
```

Comment:
Was this change intentional? All the deletions to the customAxios?

Reply:
Yes, I just updated the description to explain why I updated this file as well :)